src/Cookie/FileCookieJar.php

```diff
@@ -88,7 +89,7 @@ public function load(string $filename): void
             foreach ($data as $cookie) {
                 $this->setCookie(new SetCookie($cookie));
             }
-        } elseif (\strlen($data)) {
+        } elseif (is_string($data) && \strlen($data)) {
```

Comment:
I never updated this part for various reasons... what if the json decode above returns an `int` or a `bool` we don't have strict types enabled so this was working so far.

Reply:
Please if you have other changes like this.. review them again.

Reply:
Added unit tests to cover these values in 68f1314d52befd4f6304d368dfef2c9d17b2e244

In other words, you could say that the PHPStan error that was reported here exposed a possible bug.